src/variables/components/TypeAheadVariableDropdown.tsx

```diff
@@ -160,7 +242,7 @@ class TypeAheadVariableDropdown extends PureComponent<Props, MyState> {
     const {selectedValue} = this.props
 
     if (actualVal || selectedValue) {
-      const realValue = actualVal ?? selectedValue
+      const realValue = selectedValue ?? actualVal
```

Comment:
out of curiosity, is this change in a test? i don't remember all the e2e details

Reply:
this change is to make sure that the selectedValue trumps the previously selected value;
it involves race conditions and such; 

it is implicitly tested, since the e2e tests select one thing, then select another thing, and click away....

but adding more tests to be sure.